analysis/src/CompletionFrontEnd.ml

```diff
@@ -1184,6 +1193,34 @@ let completionWithParser1 ~currentFile ~debug ~offset ~path ~posCursor
                        fieldName;
                        posOfDot;
                        exprLoc = expr.pexp_loc;
+                       inJsx = !inJsxContext;
+                     }));
+             setFound ())
+    (* A dot completion of an integer inside braces. Useful for ->React.int completion.
+       Example: {42.}
+    *)
+    | Pexp_constant (Pconst_float (float, _))
+      when String.ends_with ~suffix:"." float
+           && Utils.hasBraces expr.pexp_attributes ->
+      let length = String.length float - 1 in
+      let integerExpr =
+        {
+          expr with
+          pexp_desc =
+            Pexp_constant (Pconst_integer (String.sub float 0 length, None));
+        }
+      in
+      exprToContextPath ~inJsxContext:!inJsxContext integerExpr
+      |> Option.iter (fun cpath ->
+             setResult
+               (Cpath
+                  (CPField
+                     {
+                       contextPath = cpath;
+                       fieldName = "";
+                       posOfDot;
+                       exprLoc = expr.pexp_loc;
+                       inJsx = !inJsxContext;
```

Comment:
Interesting, nice! Should we maybe handle this for all constants?

Reply:
Sure thing

Reply:
Removed